app/controllers/entities_controller.rb

```diff
@@ -0,0 +1,36 @@
+class EntitiesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_group
+
+  def index
+    @entities = @group.entities.order(created_at: :desc)
+    @total_amount = @entities.sum(:amount)
+  end
+
+  def new
+    @entity = Entity.new
+  end
+
+  def create
+    @entity = @group.entities.build(entity_params)
+    @entity.author_id = current_user.id
+
+    if @entity.save
+      redirect_to group_entities_path(@group), notice: 'Transaction added successfully!'
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] I signed up, and created a category and when I try to create a transaction I get errors. If we take a look at your Entity schema, there is no attribute called `author_id`, but from the project requirement there should be an `author_id` in the Entity table. Kindly look into that.

  ![image](https://github.com/rudicarrilloypr/actions-app/assets/36542986/71d3bbc5-f6bc-4c06-82a2-28ff78a18583)

  ![image](https://github.com/rudicarrilloypr/actions-app/assets/36542986/d0d6eb05-e75b-4906-a113-c1bfc29c97e0)

Reply:
<img width="647" alt="Captura de pantalla 2023-10-27 a la(s) 3 25 19" src="https://github.com/rudicarrilloypr/actions-app/assets/125106965/ebb7f28e-ff03-43b0-85be-4fab8114c475">

@TedLivist there is an author_id on the entities table, the apps works perfectly (as you can see on the sneek peak), maybe you reviewed the work branch or not install all the necessary gemfiles or bundles.

Reply:
Hi @rudicarrilloypr, I reviewed the `dev` branch you submitted. See here.

![image](https://github.com/rudicarrilloypr/actions-app/assets/36542986/36dd5557-d34c-4262-aaf3-1f8be3d00d78)

I have also gone through your migration files on this `dev` branch again, and I cannot find any migration file that adds `author_id` to the `entities`. I can only find `user_id` which is added to the schema [here](https://github.com/rudicarrilloypr/actions-app/blob/dev/db/migrate/20231023203443_create_entities.rb) but later removed [here](https://github.com/rudicarrilloypr/actions-app/blob/dev/db/migrate/20231024184411_remove_user_id_from_entities.rb)

Reply:
@TedLivist and what feedback do you think can help me to solve this? In my locals works fine, but on render it's happening the error you're saying

Reply:
@rudicarrilloypr if you want to replicate this error locally you would need to drop your dB, create it again, then run your migrations. You'll be able to reproduce this error.

To fix the issue then you would need to create a migration that adds `author_id` column to entities table, and then start fixing any resulting issues from there.

Reply:
```
rails db:drop
rails db:create
rails db:migrate
```